frontend/src/views/WheresMyGene/components/GeneSearchBar/components/SaveExport/index.tsx

```diff
@@ -227,113 +257,155 @@ export default function SaveExport({
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("png")}
-                  />
-                }
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="PNG"
+                caption="Image files that can be used on websites, have transparent backgrounds, and are suitable for graphics and digital images."
                 onChange={() => selectFileType("png")}
+                checked={selectedFileTypes.includes("png")}
                 data-testid="png-checkbox"
               />
-
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("svg")}
-                  />
-                }
+            </StyledInputCheckboxWrapper>
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="SVG"
+                caption="Vector image files that can be scaled without losing quality and are most suitable for scientific publications, presentations, and data visualization."
                 onChange={() => selectFileType("svg")}
+                checked={selectedFileTypes.includes("svg")}
                 data-testid="svg-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
+
           <StyledSection>
             <StyledTitle>DATA</StyledTitle>
-            <StyledDiv>
-              <StyledFormControlLabel
-                control={
-                  <StyledInputCheckBox
-                    checked={selectedFileTypes.includes("csv")}
-                  />
-                }
+            <StyledInputCheckboxWrapper>
+              <InputCheckbox
                 label="CSV"
+                caption="Plain text files that store tabular data by separating values with commas, making them easy to read and manipulate with spreadsheet software."
                 onChange={() => selectFileType("csv")}
+                checked={selectedFileTypes.includes("csv")}
                 data-testid="csv-checkbox"
               />
-            </StyledDiv>
+            </StyledInputCheckboxWrapper>
           </StyledSection>
-        </div>
-
-        <StyledButtonContainer>
-          <DownloadButton
-            sdsType="secondary"
-            sdsStyle="minimal"
-            onClick={handleButtonClick}
-          >
-            Cancel
-          </DownloadButton>
-          <DownloadButton
-            sdsStyle="square"
-            sdsType="primary"
-            sdsSize="large"
-            onClick={handleDownload}
-            disabled={!selectedFileTypes.length}
-            data-testid="dialog-download-button"
-          >
-            Download
-          </DownloadButton>
-        </StyledButtonContainer>
+
+          <StyledMessage>
+            <UnderlyingDataChangeBanner
+              centered={false}
```

Comment:
Decided by UX team that text should only be centered for exports

Reply:
add to code comment please!